wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
Typically the way I handle that is to create a static "constructor" method on the business Entity.  So it would handle the logic in when it can and can't be created.  Ideally it would then use a private constructor which then prevents creation of entities which are not valid.  I'm not sure if the MongoDB library we're using supports private constructors or not though, so it would be good to find that out.